apps/omg/config/config.exs

```diff
@@ -7,7 +7,8 @@ config :omg,
   ethereum_events_check_interval_ms: 500,
   coordinator_eth_height_check_interval_ms: 6_000,
   client_monitor_interval_ms: 4500,
-  metrics_collection_interval: 60_000
+  metrics_collection_interval: 60_000,
+  ws_url: {:system, "ETHEREUM_WS_RPC_URL", "ws://localhost:8546/ws"}
```

Comment:
this config entry being moved from `:omg_eth` to `:omg` doesn't look right, this kind of configuration is strictly related to the connection to the eth node's APIs, hence `:omg_eth`. Configuring of this connection should be none of `:omg`s business.

Reply:
Hmmm, yes this is confusing to me as well. The problematic part is how omg is coupled with omg_eth. On one point, omg_eth is a very weak interface towards ethereum clients, being standalone doesn’t make sense. Maybe we should consider making the integration stronger on omg_eth part and weaker on omg. That includes monitoring for example. 

Pulling application settings specific to omg_eth into omg was bothering me...

Reply:
so maybe the solution is to put :omg_eth back as the holder of :ws_url, but then seriously consider moving `EthereumClientMonitor` and `EthereumHeight` into :omg_eth. WDYT @pdobacz ?
That would separate plasma related implementation into :omg, and non specific ethereum client integration into :omg_eth

Reply:
> seriously consider moving EthereumClientMonitor and EthereumHeight into :omg_eth.

:+1: Based on my limited knowledge about these two, and their names, it sounds right to move them to `:omg_eth`. `:omg_eth` was intended to be super-minimalistic, just taking care of encodings, RPC etc. but those two might still fit, as long as they don't have any `omg`-specific logic involved.